discovery-provider/plugins/notifications/src/processNotifications/indexAppNotifications.ts

```diff
@@ -116,11 +116,13 @@ export class AppNotificationsProcessor {
     if (notifications.length == 0) return
     logger.info(`Processing ${notifications.length} push notifications`)
     const timer = new Timer('process app push notifications')
+    const blocknumber = notifications[0].blocknumber
```

Comment:
roughly something like this is how we can grab the blockhash from this number as it's possible we see the same number for reverts, seeing a different hash with the same number could prove it came from a revert though
```
const { blockhash, parenthash } = await this.dnDB('blocks').where('number', '=', blocknumber).first()
```

Reply:
ahh ok i see i just updated!